sktime/forecasting/tests/test_all_forecasters.py

```diff
@@ -931,101 +931,53 @@ def test_pridect_signature(self, estimator_class):
         assert "X" in sig.parameters.keys()
         assert "y" in sig.parameters.keys()
 
-    def test_global_forecasting_multiindex_hier(self, estimator_instance):
-        from sktime.utils._testing.hierarchical import _make_hierarchical
-
-        data_length = 100
-        data = _make_hierarchical(
-            (5, 100),
-            n_columns=2,
-            max_timepoints=data_length,
-            min_timepoints=data_length,
+    def _check_global_tag(self, estimator_instance):
```

Comment:
I am not sure if this is a good idea. 

But could we create a fixture that is only returning global forecasters. Probably would require to override generate_estimator_instance? @fkiraly what is your opinion on this?

Furthermore, I am unsure if this should be part of this PR.

Reply:
I tried to override `_generate_estimator_instance` but to find that `check_estimator` actually doesn't call it at all. 
Therefor if we want `check_estimator` to work, the tag checking has to be done inside the tests.

Reply:
Oh I See. Only the test generation is calling it. Ok in this case. I agree in merging this branch.